src/core/server/saved_objects/migrationsv2/actions/index.ts

```diff
@@ -457,16 +457,14 @@ export interface ReadWithPit {
 export const readWithPit = (
   client: ElasticsearchClient,
   pitId: string,
-  /* When reading we use a source query to exclude saved objects types which
-   * are no longer used. These saved objects will still be kept in the outdated
-   * index for backup purposes, but won't be available in the upgraded index.
-   */
-  unusedTypesQuery: Option.Option<estypes.QueryContainer>,
+  query: estypes.QueryContainer | undefined,
```

Comment:
It's always defined, so I removed the `Option` container

Reply:
> It's always defined
You mean that `query` is always defined as a parameter to `readWithPit`, but it can take on a `value` of `undefined`, right?
In that case, why wouldn't `Option` as `None` work?
[from the docs](https://gcanti.github.io/fp-ts/modules/Option.ts.html):
> If the value is absent, the Option<A> is an instance of None.

I'm just trying to understand the diff and effect that has on fp.

Reply:
> Why the | undefined then? For testing purposes?

> Was the comment meant for reindex, L586, where you changed from Option.Option<estypes.QueryContainer> to estypes.QueryContainer?

yes for both